### DISCUSSION TITLE
Can you install Data Loader in Silent Mode

### DISCUSSION THREAD

**Discussion by cjainBunnings** [forcedotcom/dataloader#1078] [Ideas]
Question: Can you install data loader in silent mode where you don't need to provide any user input. My business case is I am trying to automate some salesforce updates using data loader using azure pipelines so I need to install data loader by itself.

**Comment by ashitsalesforce** [comment]
Hi @cjainBunnings , you can install data loader on one desktop, zip the installation folder, and unzip it on other desktops. Let us know if that's a reasonable workaround.

**Comment by ashitsalesforce** [comment]
v61 will have the following command line options for install.command and install.bat to skip user prompts:
```
salesforce.installation.dir
salesforce.installation.shortcut.desktop
salesforce.installation.shortcut.windows.startmenu
salesforce.installation.shortcut.macos.appsfolder
```

For example, you can use it as:
```
install.bat  salesforce.installation.dir="c:\salesforce\dataloader\v61.0.0" salesforce.installation.shortcut.desktop=false salesforce.installation.shortcut.windows.startmenu=false
```

**Reply by xdskptv** [reply]
hi  @ashitsalesforce, how and where should I add those lines?, I'm trying to get Dataloader to install silently on MacOS
Thanks

**Comment by ashitsalesforce** [comment]
Hi @xdskptv , these are command line arguments. On MacOS you should run the following command from Terminal app:
```
install.command  salesforce.installation.dir="<path to installation folder>" salesforce.installation.shortcut.desktop=false salesforce.installation.shortcut.macos.appsfolder=false
```
